guides/common/modules/proc_exporting-templates.adoc

```diff
@@ -3,6 +3,9 @@
 
 You can export templates to a version control server, such as a Git repository.
```

Comment:
I know it was already there before, but it only supports git as a VCS. This text could imply another VCS (such as mercurial or subversion) is supported.

Reply:
Fixed.